examples/example_eval_swift_openai_api.py

```diff
@@ -2,8 +2,7 @@
 
```

Comment:
Pls update readme (and readme_zh)，and add UTs for VLMEvalKit in the tests module.

Reply:
Done, pls review again